examples/api/runner.cpp

```diff
@@ -345,6 +354,27 @@ int EXAMPLES_CDECL main(int argc, char** argv) {
             runner.set_jobs(static_cast<unsigned int>(jobs));
             set_jobs = true;
         }
+
+        // Allow disabling use of fork (e.g. disable for debugging).
```

Comment:
Consider also adding options to list components and select one component.

For example, I wanted to test changes to `api/mongocxx/examples/change_streams/with_pipeline.cpp`. Running all components against a sharded cluster took about 8.5s. So to speed up, I added manual prints and skips to identify and only run the component `api_mongocxx_examples_change_streams_with_pipeline`. Having options to list (`--list`?) and run a single component (`--component`?) may ease local development.



Though I do not think this is necessary to address in this PR. I think this suggestion is a "nice-to-have".